src/main/java/com/amazonaws/services/dynamodbv2/datamodeling/internal/Base64.java

```diff
@@ -0,0 +1,35 @@
+package com.amazonaws.services.dynamodbv2.datamodeling.internal;
+
+import java.util.Base64.Decoder;
+import java.util.Base64.Encoder;
+
+/**
+ * A class for decoding Base64 strings and encoding bytes as Base64 strings.
+ */
+public class Base64 {
```

Comment:
What value does this class add rather than using `java.util.Base64` directly?

Reply:
Updating the Base64 implementation is easier.

Reply:
If it weren't for the unusual requirement for the mimedecoder, I would say simplify the codebase by stripping this out and using the `java.util.Base64` directly. Because we have the special decoder requirement, though, this seems fine.

Reply:
Yeah, I caught myself picking the wrong decoder when I used it directly a few times. `getDecoder()` looks too reasonable, but is much more strict than the SDK1 decoder.